eng/Versions.props

```diff
@@ -141,6 +143,7 @@
     <FluentAssertionsVersion>4.19.2</FluentAssertionsVersion>
     <FluentAssertionsJsonVersion>4.19.0</FluentAssertionsJsonVersion>
     <MicrosoftDotNetXUnitExtensionsVersion>5.0.0-beta.20580.4</MicrosoftDotNetXUnitExtensionsVersion>
+    <MoqPackageVersion>4.8.2</MoqPackageVersion>
```

Comment:
This should not be the first time we use moq, could you unify other projects using moq with this version?

Reply:
There are a couple existing references to it. I updated the reference Microsoft.NET.Sdk.Publish.Tasks.Tests to use this variable. https://github.com/dotnet/sdk/blob/5e440989fcadb67a4f91dc4da993c85c927e4be5/src/Tests/dotnet.Tests/dotnet.Tests.csproj#L148 also has a reference to Moq, but uses an older version. I wasn't sure if this was oversight or intentional, so left it alone.

Reply:
could you try to make them all the same? If it fails to run we can revert it back